CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ else()
 	include_directories(dspal/include)
 	add_subdirectory(os/qurt)
 	list(APPEND df_link_libs df_qurt_stubs)
+	add_definitions(-D__PX4_QURT)
```

Comment:
We might want to rethink the name. Please join the discussion on https://github.com/PX4/DriverFramework/issues/144